Testnet panic:  datanode n08 (invalid memory address or nil pointer dereference)

Log:
```
panic: runtime error: invalid memory address or nil pointer dereference
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]: [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x18d5080]
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]: goroutine 38541394 [running]:
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]: [code.vegaprotocol.io/vega/datanode/entities.(*ID[...]).String(0x203000?)](http://code.vegaprotocol.io/vega/datanode/entities.(*ID[...]).String(0x203000?))
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]:     /home/runner/work/vega/vega/datanode/entities/id.go:90
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]: [code.vegaprotocol.io/vega/datanode/entities.(*AggregatedLedgerEntries).ToProto(0xc0051fef80)](http://code.vegaprotocol.io/vega/datanode/entities.(*AggregatedLedgerEntries).ToProto(0xc0051fef80))
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]:     /home/runner/work/vega/vega/datanode/entities/aggregated_ledgerentries.go:30 +0x54
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]: [code.vegaprotocol.io/vega/datanode/entities.AggregatedLedgerEntries.ToProtoEdge({{0x19a7e6d0](http://code.vegaprotocol.io/vega/datanode/entities.AggregatedLedgerEntries.ToProtoEdge(%7B%7B0x19a7e6d0), 0xedae49c73, 0x3db2a60}, {0xc005be5300, 0x0}, 0xc005ad4264, 0x0, 0x0, 0x0, 0x0}, ...)
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]:     /home/runner/work/vega/vega/datanode/entities/aggregated_ledgerentries.go:64 +0x33
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]: [code.vegaprotocol.io/vega/datanode/api.makeEdges[...]({0xc005214000](http://code.vegaprotocol.io/vega/datanode/api.makeEdges[...](%7B0xc005214000), 0x3e8, 0x0}, {0x0?, 0x0, 0x0})
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]:     /home/runner/work/vega/vega/datanode/api/trading_data_v2.go:1531 +0x155
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]: [code.vegaprotocol.io/vega/datanode/api.(*tradingDataServiceV2).ListLedgerEntries(0xc000491380](http://code.vegaprotocol.io/vega/datanode/api.(*tradingDataServiceV2).ListLedgerEntries(0xc000491380), {0xc0019f0348?, 0xc007d443c0?}, 0xc002044dc0)
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]:     /home/runner/work/vega/vega/datanode/api/trading_data_v2.go:260 +0x585
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]: [code.vegaprotocol.io/vega/protos/data-node/api/v2._TradingDataService_ListLedgerEntries_Handler.func1({0x2b15e10](http://code.vegaprotocol.io/vega/protos/data-node/api/v2._TradingDataService_ListLedgerEntries_Handler.func1(%7B0x2b15e10), 0xc0021b5320}, {0x23370a0?, 0xc002044dc0})
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]:     /home/runner/work/vega/vega/protos/data-node/api/v2/trading_data_grpc.pb.go:2044 +0x7b
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]: [code.vegaprotocol.io/vega/datanode/api.headersInterceptor.func1({0x2b15e10](http://code.vegaprotocol.io/vega/datanode/api.headersInterceptor.func1(%7B0x2b15e10), 0xc0021b5320}, {0x23370a0, 0xc002044dc0}, 0x2018fa0?, 0xc0095c4708)
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]:     /home/runner/work/vega/vega/datanode/api/server.go:307 +0x59d
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]: [google.golang.org/grpc.chainUnaryInterceptors.func1.1({0x2b15e10](http://google.golang.org/grpc.chainUnaryInterceptors.func1.1(%7B0x2b15e10)?, 0xc0021b5320?}, {0x23370a0?, 0xc002044dc0?})
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]:     /home/runner/go/pkg/mod/google.golang.org/grpc@v1.48.0/server.go:1129 +0x5b
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]: [code.vegaprotocol.io/vega/datanode/api.remoteAddrInterceptor.func1({0x2b15e10](http://code.vegaprotocol.io/vega/datanode/api.remoteAddrInterceptor.func1(%7B0x2b15e10), 0xc0021b52c0}, {0x23370a0, 0xc002044dc0}, 0xc0058f8ca0, 0xc007de8e80)
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]:     /home/runner/work/vega/vega/datanode/api/server.go:255 +0x637
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]: [google.golang.org/grpc.chainUnaryInterceptors.func1.1({0x2b15e10](http://google.golang.org/grpc.chainUnaryInterceptors.func1.1(%7B0x2b15e10)?, 0xc0021b52c0?}, {0x23370a0?, 0xc002044dc0?})
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]:     /home/runner/go/pkg/mod/google.golang.org/grpc@v1.48.0/server.go:1132 +0x83
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]: [google.golang.org/grpc.chainUnaryInterceptors.func1({0x2b15e10](http://google.golang.org/grpc.chainUnaryInterceptors.func1(%7B0x2b15e10), 0xc0021b52c0}, {0x23370a0, 0xc002044dc0}, 0xc0058f8ca0, 0xc0095c4708)
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]:     /home/runner/go/pkg/mod/google.golang.org/grpc@v1.48.0/server.go:1134 +0x12b
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]: [code.vegaprotocol.io/vega/protos/data-node/api/v2._TradingDataService_ListLedgerEntries_Handler({0x24b5560](http://code.vegaprotocol.io/vega/protos/data-node/api/v2._TradingDataService_ListLedgerEntries_Handler(%7B0x24b5560)?, 0xc000491380}, {0x2b15e10, 0xc0021b52c0}, 0xc0091c81c0, 0xc000fcd400)
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]:     /home/runner/work/vega/vega/protos/data-node/api/v2/trading_data_grpc.pb.go:2046 +0x138
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]: [google.golang.org/grpc.(*Server).processUnaryRPC(0xc0014f6000](http://google.golang.org/grpc.(*Server).processUnaryRPC(0xc0014f6000), {0x2b23c80, 0xc000e56820}, 0xc0024e46c0, 0xc000c477d0, 0x3d90eb0, 0x0)
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]:     /home/runner/go/pkg/mod/google.golang.org/grpc@v1.48.0/server.go:1295 +0xb2b
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]: [google.golang.org/grpc.(*Server).handleStream(0xc0014f6000](http://google.golang.org/grpc.(*Server).handleStream(0xc0014f6000), {0x2b23c80, 0xc000e56820}, 0xc0024e46c0, 0x0)
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]:     /home/runner/go/pkg/mod/google.golang.org/grpc@v1.48.0/server.go:1636 +0xa2f
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]: [google.golang.org/grpc.(*Server).serveStreams.func1.2()](http://google.golang.org/grpc.(*Server).serveStreams.func1.2())
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]:     /home/runner/go/pkg/mod/google.golang.org/grpc@v1.48.0/server.go:932 +0x98
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]: created by [google.golang.org/grpc.(*Server).serveStreams.func1](http://google.golang.org/grpc.(*Server).serveStreams.func1)
Oct 21 22:58:57 n08.testnet.vega.xyz visor[54330]:     /home/runner/go/pkg/mod/google.golang.org/grpc@v1.48.0/server.go:930 +0x28a
```

--- a/datanode/entities/aggregated_ledgerentries.go
+++ b/datanode/entities/aggregated_ledgerentries.go
@@ -27,19 +27,25 @@ type AggregatedLedgerEntries struct {
 func (ledgerEntries *AggregatedLedgerEntries) ToProto() *v2.AggregatedLedgerEntries {
 	lep := &v2.AggregatedLedgerEntries{}
 
-	partyIDString := ledgerEntries.PartyID.String()
-	if partyIDString != "" {
-		lep.PartyId = &partyIDString
+	if ledgerEntries.PartyID != nil {
+		partyIDString := ledgerEntries.PartyID.String()
+		if partyIDString != "" {
+			lep.PartyId = &partyIDString
+		}
 	}
 
-	assetIDString := ledgerEntries.AssetID.String()
-	if assetIDString != "" {
-		lep.AssetId = &assetIDString
+	if ledgerEntries.AssetID != nil {
+		assetIDString := ledgerEntries.AssetID.String()
+		if assetIDString != "" {
+			lep.AssetId = &assetIDString
+		}
 	}
 
-	marketIDString := ledgerEntries.MarketID.String()
-	if marketIDString != "" {
-		lep.MarketId = &marketIDString
+	if ledgerEntries.MarketID != nil {
+		marketIDString := ledgerEntries.MarketID.String()
+		if marketIDString != "" {
+			lep.MarketId = &marketIDString
+		}
 	}
 
 	if ledgerEntries.AccountType != nil {

